irohad/multi_sig_transactions/storage/impl/mst_storage_impl.cpp

```diff
@@ -40,7 +40,15 @@ namespace iroha {
 
   auto MstStorageStateImpl::updateOwnStateImpl(const DataType &tx)
       -> decltype(updateOwnState(tx)) {
-    return own_state_ += tx;
+    auto state_update = own_state_ += tx;
```

Comment:
= and += have the same priority. This means that compiler will decide the order of the operations.